src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java

```diff
@@ -110,13 +111,15 @@
     private long lingerMs;
     private int maxRetry;
     private long retryBackoffMs;
+    private DataConverter.BehaviorOnNullValues behaviorOnNullValues;
 
     public Builder(JestClient client) {
       this.client = client;
     }
 
     public Builder setType(String type) {
       this.type = type;
+      this.behaviorOnNullValues = DataConverter.BehaviorOnNullValues.DEFAULT;
```

Comment:
Why is `this.behaviorOnNullValues` changed in this setter for the type?

Reply:
Oops!